python/py-onnx/Portfile

```diff
@@ -55,11 +55,6 @@ if {${name} ne ${subport}} {
         port:py${python.version}-six \
         port:pybind11
 
-    if { ${python.version} < 35 } {
-        depends_lib-append \
-            port:py${python.version}-typing \
-    }
-
     if { ${python.version} >= 37 } {
```

Comment:
can you then please clean-up this if statement as well, since it's always true?

Reply:
Sure, thanks for spotting that. Done in f38a7354bfa